int32 health
int32 hunger
